bokehjs/test/unit/models/canvas/canvas.ts

```diff
@@ -8,7 +8,7 @@ describe("Canvas", () => {
   describe("should support composing layers", () => {
     it.dpr(1)("with devicePixelRatio == 1", async () => {
       const canvas = new Canvas({output_backend: "canvas", hidpi: true, styles: {width: "600px", height: "600px"}})
-      const {view: canvas_view} = await display(canvas, null)
+      const {view: canvas_view} = await display(canvas, [650, 650])
```

Comment:
What is the rationale/need for these test changes?

Reply:
`null` should be used when overflow is inevitable, in which case `null` disables overflow checks. This is not the case here, so there should be a viewport specified or left out for inferred viewport. Normally one wouldn't specify this altogether, but bare UI element don't allow viewport inference.